staging/src/k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/types_jsonschema.go

```diff
@@ -16,6 +16,26 @@ limitations under the License.
 
 package apiextensions
 
+type FieldErrorReason string
+
+const (
+	// ErrorReasonRequired is used to report required values that are not
+	// provided (e.g. empty strings, null values, or empty arrays).  See
+	// Required().
+	ErrorReasonRequired FieldErrorReason = "FieldValueRequired"
```

Comment:
I find it confusing that the enum name is "ErrorReasonRequired" but the value is "FieldValueRequired"

Reply:
The `ErrorReasonRequired` specified it is an error reason enum and indicate it's a required type. The "FieldValueRequired" is the actually reason or message user expected when the field failed validation. Does this make sense?

Reply:
I think we should make the go struct names more consistent with the string constants.

Reply:
I have updated all name to make it consistent.